BUILDING.md

```diff
@@ -0,0 +1,623 @@
+
+2019-08: Ubuntu Mate 18.04.2 LTS also works. However, the arm64 version seems not yet polished. Stellarium shows nice 8-14fps in 1680x1050 but frequently crashes. 
+
+
+## First Impressions, limitations
+The new driver seems to work pretty well in many cases, but some hardware limitations persist.
+
+* You cannot show your own landscapes or 3D sceneries with any textures larger than 2048x2048 pixels. Also the built-in "Sterngarten" scenery will not work. 
+* The Scenery3D plugin should be used with Perspective Projection only. All other projections require too much texture memory. 
+* It seems that some bug prevents OBJ planets from being rendered. Keep this option disabled!
+* If you want higher framerate, consider disabling the Satellites plugin, or at least think about reducing number of satellites to the ones you are critically interested in. Also, don't load too many minor bodies.  
+* The DSS layer seems to work for short periods, then the X server fails. It may have to do with limitations of texture memory. You can try to press Ctrl-Alt-Backspace to restart it, but sometimes you will have to reboot if the graphics failed.
+
+***
+
+## Building the external libraries
+
+The following instructions which are required for Raspbian "Stretch" have been taken from Eric Anholt's site (https://github.com/anholt/mesa/wiki/VC4-complete-Raspbian-upgrade) on February 24, 2018. Fortunately we don't need to build a new kernel!  
+
+
+### libdrm
+
+Until Raspbian gets its libdrm updated, we need to build it ourselves.
+
+    sudo apt-get install \
+        xutils-dev libpthread-stubs0-dev \
+        automake autoconf libtool git
+    git clone git://anongit.freedesktop.org/mesa/drm
+    cd drm
+    ./autogen.sh \
+        --prefix=/usr \
+        --libdir=/usr/lib/arm-linux-gnueabihf
+    make
+    sudo make install
+
+### Mesa
+    sudo apt-get install \
+        flex bison python-mako \
+        libxcb-dri3-dev libxcb-dri2-0-dev \
+        libxcb-glx0-dev libx11-xcb-dev \
+        libxcb-present-dev libxcb-sync-dev \
+        libxshmfence-dev \
+        libxdamage-dev libxext-dev libxfixes-dev \
+        x11proto-dri2-dev x11proto-dri3-dev \
+        x11proto-present-dev x11proto-gl-dev \
+        libexpat1-dev libudev-dev gettext \
+        libxrandr-dev
+    
+    git clone git://anongit.freedesktop.org/mesa/mesa
+    cd mesa
+    ./autogen.sh \
+        --prefix=/usr \
+        --libdir=/usr/lib/arm-linux-gnueabihf \
+        --with-gallium-drivers=vc4 \
+        --with-dri-drivers= \
+        --with-egl-platforms=x11,drm
+    make
+    sudo make install
+
+## Notes for building Stellarium
+
+You may want to install a swap file, otherwise you may run out of virtual memory when compiling
+[https://www.cyberciti.biz/faq/linux-add-a-swap-file-howto/](https://www.cyberciti.biz/faq/linux-add-a-swap-file-howto/)
+
+Then, make sure to install the [build requirements](#lbuild-dependencies) and follow the [building instructions for Linux](#linux).
```

Comment:
I randomly clicked on a few hyperlinks in an MD editor. This failed, program crashed.

Reply:
ok. what editor was that, please?

Reply:
it's possibly the GMD syntax that gives this issue.

Reply:
"Markdown Editor" by Poppen's Lab, my first usable contact with MD. I accept recommendations for better tools as long as they are free.